packages/utils/src/lib/git.integration.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from 'vitest';
+import { getLatestCommit } from './git';
+
+const gitCommitDateRegex =
+  /^(Mon|Tue|Wed|Thu|Fri|Sat|Sun) (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) \d{1,2} \d{2}:\d{2}:\d{2} \d{4} [+|-]\d{4}$/;
+
+describe('getLatestCommit', () => {
+  it('should log latest commit', async () => {
+    await expect(getLatestCommit()).resolves.toEqual(
+      expect.objectContaining({
+        hash: expect.stringMatching(/^[0-9a-f]+$/),
```

Comment:
So we still check it has the full hash length:

```suggestion
        hash: expect.stringMatching(/^[0-9a-f]{40}$/),
```